packages/fx-core/resource/package.nls.json

```diff
@@ -419,6 +419,9 @@
   "error.generator.FetchZipFromUrlError": "Unable to download zip package from %s.",
   "error.generator.UnzipError": "Unable to unzip templates and write to disk.",
   "error.generator.MissKeyError": "Unable to find key %s",
+  "error.generator.DownloadSampleApiLimitError": "Unable to download sample due to rate limit exceeded. Please wait for the rate limit reset and try again (This may take up to 1 hour). Alternatively, you can go to %s to git clone the repo manually",
```

Comment:
rate limit exceeded -> throttling

Please wait ... and try again (....) -> Retry later (....)